nanoservice/client.py

```diff
@@ -42,14 +44,18 @@ def __init__(self, addr, encoder=None, socket=None,
         self.sock.connect(self.addr)
         self.authenticator = Authenticator(secret, digestmod) if auth else None
 
-    def build_payload(self, method, args):
+    @classmethod
```

Comment:
Yes just because of pylint.
When you will change it to a instance method you could just remove the @classmethod at the same time :)